src/ToolBox/superpmi/superpmi-shared/spmidumphelper.cpp

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+//----------------------------------------------------------
+// SpmiDumpHelper.cpp - a helper to dump structs that are used in JitEEInterface calls and spmi collections.
+//----------------------------------------------------------
+
+#include "standardpch.h"
+#include "spmidumphelper.h"
+
+std::string SpmiDumpHelper::DumpAgnostic_CORINFO_RESOLVED_TOKENin(
+    const MethodContext::Agnostic_CORINFO_RESOLVED_TOKENin& tokenIn)
+{
+    char buffer[MAX_BUFFER_SIZE];
+    sprintf(buffer, "tc - % 016llX ts - % 016llX tok - %08X tt - %u", tokenIn.tokenContext, tokenIn.tokenScope,
+            tokenIn.token, tokenIn.tokenType);
```

Comment:
Does it work to have a space between `%` and `016llX`? IMO, you should remove the space.